mock/src/app.ts

```diff
@@ -0,0 +1,243 @@
+/**
+ * MOCK ONLY USED WHEN MBI API IS NOT AVAILABLE
+ * TO SIMULATE API ANSWER
+ */
+import express, { Router } from 'express';
+import cors from 'cors';
+import { MBIResourceType } from '../../src/@types/centreonAPI';
+
+function getRandomArbitrary(min: number, max: number) {
+  return Math.round(Math.random() * (max - min) + min);
```

Comment:
As we had a security issue with Math.random before, we tend to use The Web Crypto API : https://developer.mozilla.org/en-US/docs/Web/API/Crypto/getRandomValues

Reply:
this app run in node.js . getRandomValues doesn't exist in crypto https://nodejs.org/api/crypto.html#cryptorandombytessize-callback . 

Also, this part is only a mock to return random values to the grafana plugin, so nothing important + replacing it seems to be a "complex" code, only to get a random number between two values .